README.md

```diff
@@ -21,38 +21,39 @@ Features:
-| Starlark               | [Buildifier]          |                      |
-| Swift                  | [SwiftFormat] (1)     |                      |
-| TSX                    | [Prettier]            | [ESLint]             |
-| TypeScript             | [Prettier]            | [ESLint]             |
-| YAML                   | [yamlfmt]             |                      |
+| Language               | Formatter             | Linter(s)                        |
+| ---------------------- | --------------------- |----------------------------------|
+| C / C++                | [clang-format]        | [clang-tidy]                     |
+| Cuda                   | [clang-format]        |                                  |
+| CSS, Less, Sass        | [Prettier]            | [Stylelint]                      |
+| Go                     | [gofmt] or [gofumpt]  |                                  |
+| GraphQL                | [Prettier]            |                                  |
+| HCL (Hashicorp Config) | [terraform] fmt       |                                  |
+| HTML                   | [Prettier]            |                                  |
+| JSON                   | [Prettier]            |                                  |
+| Java                   | [google-java-format]  | [pmd] , [Checkstyle], [Spotbugs] |
+| JavaScript             | [Prettier]            | [ESLint]                         |
+| Jsonnet                | [jsonnetfmt]          |                                  |
+| Kotlin                 | [ktfmt]               | [ktlint]                         |
+| Markdown               | [Prettier]            | [Vale]                           |
+| Protocol Buffer        | [buf]                 | [buf lint]                       |
+| Python                 | [ruff]                | [flake8], [ruff]                 |
+| Rust                   | [rustfmt]             |                                  |
+| SQL                    | [prettier-plugin-sql] |                                  |
+| Scala                  | [scalafmt]            |                                  |
+| Shell                  | [shfmt]               | [shellcheck]                     |
+| Starlark               | [Buildifier]          |                                  |
+| Swift                  | [SwiftFormat] (1)     |                                  |
+| TSX                    | [Prettier]            | [ESLint]                         |
+| TypeScript             | [Prettier]            | [ESLint]                         |
+| YAML                   | [yamlfmt]             |                                  |
 
 [prettier]: https://prettier.io
 [google-java-format]: https://github.com/google/google-java-format
 [flake8]: https://flake8.pycqa.org/en/latest/index.html
 [pmd]: https://docs.pmd-code.org/latest/index.html
 [checkstyle]: https://checkstyle.sourceforge.io/cmdline.html
+[spotbugs]: https://spotbugs.github.io/
```

Comment:
nit: since spotbugs is a fork of findbugs, and I tried to rewrite most of those checks in errorprone, and that's built-in to Bazel's Java compiler, I wonder how much overlap there is for users between spotbugs and Bazel's `javac`?